service/controller/v5/blobclient/blob_client.go

```diff
@@ -0,0 +1,186 @@
+package blobclient
+
+import (
+	"context"
+	"fmt"
+	"io/ioutil"
+	"net/url"
+	"strings"
+
+	"github.com/giantswarm/microerror"
+
+	"github.com/Azure/azure-sdk-for-go/services/storage/mgmt/2018-07-01/storage"
+	"github.com/Azure/azure-storage-blob-go/azblob"
+)
+
+const (
+	blobFormatString      = `https://%s.blob.core.windows.net`
+	resourceNotFoundError = "NotFound"
+	maxRetriesRequests    = 3
+)
+
+type Config struct {
+	ContainerName         string
+	GroupName             string
+	StorageAccountName    string
+	StorageAccountsClient *storage.AccountsClient
+}
+
+type BlobClient struct {
+	containerName         string
+	groupName             string
+	storageAccountName    string
+	storageAccountsClient *storage.AccountsClient
+
+	// containerURL is configured separately from the default
+	// parameters.
+	containerURL azblob.ContainerURL
+}
+
+func (c *BlobClient) Boot(ctx context.Context) error {
+	var containerURL azblob.ContainerURL
+
+	key, err := c.getAccountPrimaryKey(ctx)
+	if err != nil {
+		return microerror.Mask(err)
+	}
+
+	sc, err := azblob.NewSharedKeyCredential(c.storageAccountName, key)
+	if err != nil {
+		return microerror.Mask(err)
+	}
+
+	p := azblob.NewPipeline(sc, azblob.PipelineOptions{})
+	u, _ := url.Parse(fmt.Sprintf(blobFormatString, c.storageAccountName))
```

Comment:
I suggest removing this const.

```suggestion
	u, _ := url.Parse(fmt.Sprintf("https://%s.blob.core.windows.net", c.storageAccountName))
```